CalculatorTest.py

```diff
@@ -24,5 +24,9 @@ def test_run_operation_invalid(self):
         with self.assertRaises(Exception):
             Calculator.run_operation(9,0, '8')
 
+    def test_run_operation_exponentiation(self):
```

Comment:
Can we add a few tests that use decimal numbers and negative numbers?

Reply:
I'm not sure if we need additional test cases as the current tests cover how our code uses exponentiation. I believe adding additional test cases would test the built-in functionality of exponentiation rather than how our application uses it. However, if we think they are still necessary, I can add them.

Reply:
Great!